### ISSUE TITLE
Wifi doesn't work (Cubietruck)

### ISSUE THREAD

**Issue by Daniel-Medeiros** [cubieplayer/Cubian#379]
i don't know how to make it work or it really doesn't show at the bottom only wired connection.

**Comment by nocolour** [comment]
I also facing same problem. My cubietruck wifi not working on Cubian X1 desktop. 
module bcmdhd was loaded, kernel version sun7i-3.4.79-4cubian. Network manager can not detect any wifi signal. Still trying. it should very simple and straight forward but not working.

michalliu, pls help us solve this problem. Thanks!

**Comment by Daniel-Medeiros** [comment]
yes no colour mine is also cubietruck and i have all drivers up to date and still got some more issues hopefully fixes soon :)

**Comment by nocolour** [comment]
My wireless finally working ready..  After few try. I believe cubian x1 for cubietruck have bug on wireless. It can not auto scan the SSID.

How to make the wireless working...?

I do this:

be careful right click & left click, try follow below step:

1.) disconnect the network cable
2.) right click the network manager applet "Disable wireless"
3.) right click the network manager applet "Disable Networking"
4.) right click the network manager applet "Enable Networking"
5.) right click the network manager applet "Enable wireless"
6.) sudo /etc/init.d/networking restart
7.) sudo iwlist wlan0 scan
8.) left click the network manager, to check any Wireless SSID showing or NOT
you can repeat the step 2 - 5 again. still can not, follow bellow step.

If still can not detect any SSID do bellow step:

9.) LEFT click the network manager, click "create New wireless network".
10.) insert your network name "wireless ssid" and key, than click create. 
*_**ignore the security type. *_\* BUG **\* Don't have WPA. It only have WEP only, if security is WPA, fail message will display. Ignore it. Close everything.
11.) left click the network manager, check again. The SSID will show out. Click the SSID you want connect.

12.)sudo iwlist wlan0 scan

We have do something to force the wireless scan for SSID. Don't know why it can not auto scan by default. After do above, it working fine.

Cubian team need to verify the problem, wireless SSID can not auto scan and not show on Mate desktop network manager !

Thanks!

**Comment by Daniel-Medeiros** [comment]
please Cubian team fix this issue it is kinda annoying to have wireless by lucky lol

**Comment by michalliu** [comment]
I didn't experience such a problem personally, need more reports on this issue

**Comment by jjaing** [comment]
I have the same problem, the wifi can not detect any SSID

**Comment by albig** [comment]
I have the problem, too. No networks are shown in the network manager.

I can workaround it, by manually "connect to a hidden network". After entering the SSID and the WPA password , it connects quite fast.

After connection, it shows available networks in the neighbourhood ;-)

cubian for desktop on cubietruck

**Comment by gordol** [comment]
does this fix issue for you guys?
ifconfig p2p0 down
ifconfig wlan0 down
ifconfig wlan0 up

then suddenly networks show up in network-manager

**Comment by wspirit** [comment]
how to connect automatically to wifi?
